client/src/routes/root.jsx

```diff
@@ -9,10 +9,25 @@ const Root = () => {
   const [played, setPlayed] = useState('');
 
   function getDate() {
+    const norskeMåneder = [
+      'Januar',
+      'Februar',
+      'Mars',
+      'April',
+      'Mai',
+      'Juni',
+      'Juli',
+      'August',
+      'September',
+      'Oktober',
+      'November',
+      'Desember',
+    ];
+
     try {
       const today = new Date();
       const day = today.getDate();
-      const month = today.toLocaleString('default', { month: 'long' });
+      const month = norskeMåneder[today.getMonth()];
```

Comment:
Ref. tidligere kommentar.

```suggestion
      const month = norwegianMonths[today.getMonth()];
```